packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -49,18 +49,67 @@ export const GET_ACCOUNT_BY_EMAIL = gql`
   }
 `;
 
+// new query for getting account by UID
+export const GET_ACCOUNT_BY_UID = gql`
+  query getAccountByUid($uid: String!) {
+    accountByUid(uid: $uid) {
+      uid
+      createdAt
+      emails {
+        email
+        isVerified
+        isPrimary
+        createdAt
+      }
+      emailBounces {
+        email
+        createdAt
+        bounceType
+        bounceSubType
+      }
+    }
+  }
+`;
+
+function validateUID(uid: string) {
+  // checks if input string is in uid format
+  if (/^[0-9a-fA-F]{32}/.test(uid)) {
+    // alphanumeric, 32 digit
+    return true;
+  }
+  return false;
```

Comment:
this can just be `return /^[0-9a-fA-F]{32}/.test(uid)` since the result of `test` is a boolean